src/postgres-util/Cargo.toml

```diff
@@ -12,6 +12,7 @@ mz-cloud-resources = { path = "../cloud-resources", optional = true }
 mz-ore = { path = "../ore", features = ["async"], optional = true }
 mz-proto = { path = "../proto", optional = true }
 mz-repr = { path = "../repr", optional = true }
+mz-secrets = { path = "../secrets" }
```

Comment:
This crate was intentionally `mz-secrets` agnostic.

Reply:
The only way to get rid of this is if all users of this crate pre-fetch the secret before any usage of the `SshTunnelManager`, which is probably fine?

Reply:
Yes, exactly! Not just fine but desirable, IMO.